server/logger.js

```diff
@@ -0,0 +1,100 @@
+import winston from 'winston';
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+const { combine, timestamp, printf, colorize } = winston.format;
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const ROOT = __dirname;
+
+// Apply colors to log levels
+const colorizer = colorize({ all: true });
```

Comment:
The export statement now includes `socket_user_map`, which is a good addition. Ensure that the documentation is updated to reflect this change, so other developers are aware of the new export.


    *This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*